src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -203,30 +90,6 @@ public static void RecordException(this Activity activity, Exception ex)
             activity?.AddEvent(new ActivityEvent(SemanticConventions.AttributeExceptionEventName, default, tagsCollection));
         }
 
-        private struct ActivitySingleTagEnumerator : IActivityEnumerator<KeyValuePair<string, object>>
-        {
-            public object Value;
-
-            private readonly string tagName;
-
-            public ActivitySingleTagEnumerator(string tagName)
-            {
-                this.tagName = tagName;
-                this.Value = null;
-            }
-
-            public bool ForEach(KeyValuePair<string, object> item)
-            {
-                if (item.Key == this.tagName)
-                {
-                    this.Value = item.Value;
-                    return false;
-                }
-
-                return true;
-            }
-        }
-
         private struct ActivityStatusTagEnumerator : IActivityEnumerator<KeyValuePair<string, object>>
```

Comment:
@cijothomas Did you mean to leave this here? Seems like it is also in the new file.

Reply:
missed this! thanks!